contracts/Events/EventManager.sol

```diff
@@ -10,6 +9,11 @@ contract OutcomeToken is MintableToken, StandardBurnableToken {}
 contract EventManager is OracleConsumer {
     using SafeMath for uint;
 
+    event EventCreation(bytes32 indexed questionId);
+    event EventResolution(bytes32 indexed questionId, bytes result);
```

Comment:
Adding these events is a good idea. I wonder if it would be better to parse the `uint[] payouts` out from the result and present that in the event instead of the unformatted result. On the one hand, this would be more processing work for the smart contract; something which clients can handle. On the other hand, it makes these events more "correct" in some sense, in that clients would more readily know how to interpret a `uint[] payouts` array, whereas `bytes result` would take some explaining for clients to get.